Dockerfile

```diff
@@ -1,6 +1,8 @@
 FROM ruby:2.6.6-alpine as build-stage
 
 ENV RAILS_ENV=production
+ENV RACK_ENV=production
+
```

Comment:
We probably don't need either (I know I added the `RAILS_ENV` one) at the build stage, do we?
Are these ENV vars considered during `bundle install`?  🤔

Reply:
Maybe we should remove/investigate them separately.